openshift/ci-operator/build-image/Dockerfile

```diff
@@ -13,14 +13,11 @@
 # limitations under the License.
 
 # Dockerfile to bootstrap build and test in openshift-ci
-FROM openshift/origin-release:golang-1.16
+FROM registry.ci.openshift.org/openshift/release:golang-1.16
 
-# Add kubernetes repository
-ADD openshift/ci-operator/build-image/kubernetes.repo /etc/yum.repos.d/
```

Comment:
That should be handled by CI infra steps already.

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/knative-client/openshift-knative-client-release-next__48.yaml#L136